### DISCUSSION TITLE
M1 MacBook Pro 配置简述

### DISCUSSION THREAD

**Discussion by giscus** [ZexWoo/Giscus#18] [Announcements]
# M1 MacBook Pro 配置简述

买了个 MacBook Pro，因为系统不熟悉的缘故，费了不少劲，这里做下记录，方便以后回看。

https://zexwoo.blog/posts/tutorials/configure-m1-mac/

**Comment by ZHH-start** [comment]
看起来Mac是真的难用。好吧，成功劝退了。我到现在都没法理解为啥有人喜欢Mac-因为我tm**被Ubuntu折腾得快死了！！**
